assets/js/event-filtering.js

```diff
@@ -0,0 +1,67 @@
+$(function() {
+    var eventsElements = $(".event-tags span");
```

Comment:
What does jQuery return if no matches are found? Does it return `[]` or `undefined`? Similarly, what about `$("#eventFilter")` below? (In that case, I assume it's a single object?)

In either case, it seems like we should add some basic checks to ensure the script doesn't fail at runtime. Right?

Reply:
They return `[]` if it does not exist. I added checks to make sure the element returns before proceeding, to err on the side of caution. They are not really needed because nothing will happen if `eventsElements` comes back empty.

https://learn.jquery.com/using-jquery-core/faq/how-do-i-test-whether-an-element-exists/